lib/modules/manager/gomod/line-parser.ts

```diff
@@ -17,6 +17,10 @@ const replaceRegex = regEx(
   /^(?<keyword>replace)?\s+(?<module>[^\s]+\/[^\s]+)\s*=>\s*(?<replacement>[^\s]+)(?:\s+(?<version>[^\s]+))?(?:\s*\/\/\s*(?<comment>[^\s]+)\s*)?$/,
 );
 
+export const excludeBlockStartRegex = regEx(/^(?<keyword>exclude)\s+\(\s*$/);
+
+export const endBlockRegex = regEx(/^\s+\)\s*$/);
```

Comment:
This is pretty generic. Is there any chance of a false positive or are exclude blocks never nested or nesting?